--- libpng/mkfile	2015-05-28 20:56:05.499222377 +0300
+++ libpng/mkfile.new	2015-05-28 20:56:16.487357159 +0300
@@ -2,5 +2,5 @@
 
 <v.mk
 src = libpng-$v
-url = http://optimate.dl.sourceforge.net/project/libpng/libpng16/$v/libpng-$v.tar.xz
+url = http://ftp.lfs-matrix.net/pub/blfs/conglomeration/libpng/libpng-$v.tar.xz
 mkfile = ../libpng.mk
--- libpng/libpng.mk	2015-04-05 10:54:07.398139482 +0300
+++ libpng/libpng.mk.new	2015-05-30 08:10:34.043460558 +0300
@@ -1,7 +1,7 @@
 # TODO: build and package binaries? png2pnm, pngfix, libpng-config etc?
 
 LIB = libpng.a
-DEPS = zlib
+DEPS = zlib xz
 BIN = \
 	contrib/pngminus/pnm2png \
 	contrib/pngminus/png2pnm
